.eslintrc.json

```diff
@@ -74,6 +74,13 @@
 		"no-negated-in-lhs": "error",
 		"no-nested-ternary": "error",
 		"no-redeclare": "error",
+		"no-restricted-syntax": [
+			"error",
+			{
+				"selector": "ImportDeclaration[source.value=/^@wordpress\\u002F.+\\u002F/]",
```

Comment:
What happens when we begin requiring `npm` packages like `@wordpress/hooks`?  Path imports should be fine there.

Do we want to match our list of Webpack entry points specifically, or keep this restriction for all `@wordpress` imports for consistency?